### ISSUE TITLE
Startup fails if not running in AWS environment

### ISSUE THREAD

**Issue by posto** [remind101/ssm-env#28]
Starting a docker image with ssm-env 0.4 fails outside the AWS environment.

This looks like a regression, as ssm-env 0.3 starts fine. 

```
$ docker run -it --rm mydockerimage:latest
panic: EC2MetadataRequestError: failed to get EC2 instance identity document
caused by: RequestError: send request failed
caused by: Get http://169.254.169.254/latest/dynamic/instance-identity/document: dial tcp 169.254.169.254:80: connect: no route to host

goroutine 1 [running]:
github.com/aws/aws-sdk-go/aws/session.Must(...)
        /go/pkg/mod/github.com/aws/aws-sdk-go@v0.0.0-20170228030727-78568b07950e/aws/session/session.go:232
main.main()
        /go/src/github.com/remind101/ssm-env/main.go:70 +0x396
```

Relevant Dockerfile content:
```
ENTRYPOINT ["/usr/local/bin/ssm-env", "-with-decryption"]
CMD ["java","-Djava.security.egd=file:/dev/./urandom","-jar","/app.jar"]
```

**Comment by russellballestrini** [comment]
ooph, this is likely because I merged this: https://github.com/remind101/ssm-env/pull/14 @posto do you have any suggestions on how we could guard this?

**Comment by russellballestrini** [comment]
I think as a work around, if you set the `AWS_REGION` it will not error.

**Comment by cojack** [comment]
@russellballestrini same here

**Comment by russellballestrini** [comment]
@cojack did you try the work around? (set `AWS_REGION`)

**Comment by alexions** [comment]
I can confirm, that this issue is caused by missing `AWS_REGION` env var. By setting this var to some value, like `eu-west-1` the issue disappears.

**Comment by russellballestrini** [comment]
@alexions Do you have any suggestions on how we could "fix" this instead of relying on the work around?

**Comment by alexions** [comment]
My proposal is here: https://github.com/remind101/ssm-env/pull/36

**Comment by russellballestrini** [comment]
Fixed in #36

Thanks @alexions !!
